### PR TITLE
Add PSR-21 to index and CODEOWNERS

### PR DESCRIPTION
**Pull request by vdelau**

### CODE REVIEW

**Review by samdark — APPROVED** [review]

**Review by drupol — APPROVED** [review]
LGTM

File: CODEOWNERS [hunk 1]
```diff
@@ -2,3 +2,4 @@
 /proposed/security* @php-fig/psr-9-10
 /proposed/phpdoc* @php-fig/psr-5
 /proposed/clock* @php-fig/psr-20
+/proposed/internationalization* @php-fig/psr-21
```
Comment (Jean85) [inline CODEOWNERS:5]: This group has yet to be created here: https://github.com/orgs/php-fig/teams

**Review by Jean85 — CHANGES_REQUESTED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (vdelau) [reply CODEOWNERS:5]: Group is already created and members added

**Review by Jean85 — APPROVED** [review]

**Comment by vdelau** [comment]
Thanks, I've updated the website as well

**Pull request merged**